architecture-decision-records/0021-service-unavailable-page.md

```diff
@@ -0,0 +1,39 @@
+# 20. Deploying a service unavailable page 
+
+**Date:** 2021-07-14
+
+## Context
+We need a mechanism to manually deploy a service unavailable page. There are two main reasons we might want to do this. 
+* There is a serious issue with the site that we can't fix immediately.
+* We want to carry out maintenance which will affect the service. 
+
+The page will be a single static html page which will be displayed when a user goes to any of the https://tdr.nationalarchives.gov.uk/* urls.
+
+![Service Unavailable Design](images/service-unavailable-design.png)
+
+## Evaluation criteria
+* We must be able to deploy the page manually.
+* We should avoid creating any permanently running services for this page as it will be used rarely.
+* The switch to the new page should only take seconds.
+* It should be easy for anyone on the TDR team to deploy.
+* It should be easy to switch back to the TDR app.
+
+## Options considered
+
+### Options for rendering the page
+![Page rendering options](images/failover-page-options.png)
+
+### Options for carrying out the switchover
+![Route 53 Failover](images/route-53-failover.png)
```

Comment:
Could you copy-paste the titles and the advantage/disadvantage lists into the text of the ADR? It makes it more searchable and accessible.

Reply:
Good point. I've pasted in the notes from the images into the main text.